components/category-select-table.vue

```diff
@@ -2,9 +2,9 @@
 
     <div>
 
-        <b-row class="no-padding-row">
+        <b-row>
```

Comment:
not sure what this does, mind checking again that this was intentional?

Reply:
I checked when I was working on the layout and there is no built-in `no-padding-row` class in Bootstrap and we don't implement it in the code base either.